aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        h_distance = cuspatial.haversine_distance(df['pickup_longitude'], df['pickup_latitude'], df['dropoff_longitude'], df['dropoff_latitude'])
</pre><p>This is not very important , but aren't these supposed to be the lat long values which you just round to 2 decimal places?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Are you talking about the haversine_distance function?</p>

Reply:
It could use either. I got this function from the example on https://github.com/rapidsai-community/notebooks-contrib This would yield a more accurate distance. I'm not sure why they computed the round-up values, probably to add features for RF.

Reply:
I guess leave it then. Does not change anything in the flow anyway.